### DISCUSSION TITLE
Error while transferring files to kindle through WordDumb. PLS Help !!!

### DISCUSSION THREAD

**Discussion by Rudy-alt** [xxyzz/WordDumb#255] [Q&A]
So, my issue has been this, I've been trying on sending a spanish pdf book I converted to kfx through WordDumb to my kindle to access it's full features but so far nothing has been working for in terms of sending, it keeps giving me the same error while trying to convert it. How can I solve this ??
![Screenshot 2024-08-27 195403](https://github.com/user-attachments/assets/a988fde8-546a-409a-af93-4e8b6570368c)
![Screenshot 2024-08-27 195410](https://github.com/user-attachments/assets/06d60b55-6aac-4292-a3d8-6fdaee45b955)
![Screenshot 2024-08-27 195439](https://github.com/user-attachments/assets/783928e3-0bb7-437b-ae82-1f09abe1de27)

**Comment by xxyzz** [comment]
The book doesn't have supported language metadata, you must change language manually to Spanish in calibre's "edit metadata" button. 

But I don't think PDF file could be converted to any readable ebook format.

**Reply by Rudy-alt** [reply]
Oh I see, I did, for me it works, I converted it to kfx from pdf without issues

**Reply by Rudy-alt** [reply]
Ok, I got everything working, wordwise and x-ray, but there's only one issue, and that would be wordwise being on english, is there a way to correct this error ?
![WhatsApp Image 2024-08-27 at 22 01 17_d70a0efa](https://github.com/user-attachments/assets/bd0f6794-134a-43a3-85dc-170acaf83098)

**Reply by xxyzz** [reply]
Must change word wise language to Chinese on Kindle if Wiktionary data are used.

**Reply by Rudy-alt** [reply]
> Must change word wise language to Chinese on Kindle if Wiktionary data are used.

Worked perfectly, thank you!

**Reply by Rudy-alt** [reply]
Oh I see, thank you very much
One more question, should I "enable all" for wikitionary and wordwise for language ?
![Screenshot 2024-08-27 224408](https://github.com/user-attachments/assets/715d4a22-4ebf-4843-9e31-b0403990b3bb)

**Reply by xxyzz** [reply]
It's up to you, but you may want to disable some commonly used but simple words.
